numpy/core/src/umath/loops_trigonometric.dispatch.c.src

```diff
@@ -161,6 +253,34 @@ simd_sin_poly_f64(npyv_f64 r, npyv_u64 ir, npyv_u64 sign)
     return npyv_reinterpret_f64_u64(npyv_xor_u64(npyv_xor_u64(npyv_reinterpret_u64_f64(y), sign), odd));
 }
 
+#if NPY_SIMD_CVT_F64
+NPY_FINLINE void
+simd_tan_f64(const double *src, npy_intp ssrc, double *dst, npy_intp sdst, npy_intp len)
+{
+    const int vstep = npyv_nlanes_f64;
+
+    npyv_f64 out = npyv_zero_f64();
+    npyv_f64 x_in;
+
+    for (; len > 0; len -= vstep, src += ssrc*vstep, dst += sdst*vstep) {
+        if (ssrc == 1) {
+            x_in = npyv_load_tillz_f64(src, len);
+        } else {
+            x_in = npyv_loadn_tillz_f64(src, ssrc, len);
+        }
+
+        out = simd_inner_tan(x_in);
```

Comment:
```suggestion
#if defined(NPY_HAVE_AVX512_SKX) && defined(NPY_CAN_LINK_SVML)
       out = __svml_tan8(x_in);
#else
        out = simd_inner_tan(x_in);
#endif
```

SVML provides higher accuracy under 1ULP, I think we should keep it enabled until we find a better replacement.

Reply:
It would be good to decide whether or not to integrate a library per platform, or go the intrinsics route, albeit the routines will likely change slightly. This reply implies that we'd want to keep SVML and use third party libraries even if we had a universal variant of functions?

If you're proposing continuing with SVML, therefore continuing to rely on third party libraries, then I don't see why we can't go with something like SLEEF (https://github.com/numpy/numpy/issues/23068) or another library for AArch64? It'd cut down the amount of effort we have to put in converting all these routines 😸

Reply:
SVML isn't accurate up-to 1 ULP. It max ULP err is 3.9 and has a general ULP err distribution very similar to this patch. See https://github.com/numpy/numpy/pull/23603#pullrequestreview-1437337285. I don't see the point of keeping the SVML code.